concepticondata/conceptlists/Strong-1911-150.tsv

```diff
@@ -0,0 +1,151 @@
+Strong-1911-150-78	78	Lip	478	LIP
+Strong-1911-150-79	79	Live	1422	BE ALIVE
+Strong-1911-150-80	80	Louse	1392	LOUSE
+Strong-1911-150-81	81	Man	1554	MAN
+Strong-1911-150-82	82	Mat	195	MAT
+Strong-1911-150-83	83	Milk	635	MILK
+Strong-1911-150-84	84	Moon	1313	MOON
+Strong-1911-150-85	85	Morning	1339	MORNING
+Strong-1911-150-86	86	Mosquito	1509	MOSQUITO
+Strong-1911-150-87	87	Mother	1216	MOTHER
+Strong-1911-150-88	88	Mouth	674	MOUTH
+Strong-1911-150-89	89	Name	1405	NAME
+Strong-1911-150-90	90	Navel	1838	NAVEL
+Strong-1911-150-91	91	Near	1942	NEAR
+Strong-1911-150-92	92	Night	1233	NIGHT
+Strong-1911-150-93	93	Nipple	796	NIPPLE
+Strong-1911-150-94	94	No	1269	NO
+Strong-1911-150-95	95	Nose	1221	NOSE
+Strong-1911-150-96	96	Paddle	234	PADDLE
+Strong-1911-150-97	97	Pig	1337	PIG
+Strong-1911-150-98	98	Rain	2108	RAINING OR RAIN
+Strong-1911-150-99	99	Rat	1490	RAT
+Strong-1911-150-100	100	Red	156	RED
+Strong-1911-150-101	101	River	666	RIVER
+Strong-1911-150-102	102	Road	667	ROAD
+Strong-1911-150-103	103	Root	670	ROOT
+Strong-1911-150-104	104	Rope	1218	ROPE
+Strong-1911-150-105	105	Sago palm	2372	SAGO PALM
+Strong-1911-150-106	106	Salt	1274	SALT
+Strong-1911-150-107	107	Sand	671	SAND
+Strong-1911-150-108	108	Scratch	1436	SCRATCH
+Strong-1911-150-109	109	Sea	1474	SEA
+Strong-1911-150-110	110	See	1409	SEE
+Strong-1911-150-111	111	Shadow	1388	SHADE
+Strong-1911-150-112	112	Shark	1110	SHARK
+Strong-1911-150-113	113	Shield	1064	SHIELD
+Strong-1911-150-114	114	Sit	1416	SIT
+Strong-1911-150-115	115	Skin	763	SKIN
+Strong-1911-150-116	116	Sky	1732	SKY
+Strong-1911-150-117	117	Sleep	1585	SLEEP
+Strong-1911-150-118	118	Small	1246	SMALL
+Strong-1911-150-119	119	Smoke	778	SMOKE (EXHAUST)
+Strong-1911-150-120	120	Snake	730	SNAKE
+Strong-1911-150-121	121	Soft	1856	SOFT
+Strong-1911-150-122	122	Speak	1623	SPEAK
+Strong-1911-150-123	123	Spear	945	SPEAR
+Strong-1911-150-124	124	Spit	1440	SPIT
+Strong-1911-150-125	125	Stand	1442	STAND
+Strong-1911-150-126	126	Star	1430	STAR
+Strong-1911-150-127	127	Stone	857	STONE
+Strong-1911-150-128	128	Sun	1343	SUN
+Strong-1911-150-129	129	Sweet potato	159	SWEET POTATO
+Strong-1911-150-130	130	Taro	351	TARO
+Strong-1911-150-131	131	Taste	21	TASTE (SOMETHING)
+Strong-1911-150-132	132	Thick	1244	THICK
+Strong-1911-150-133	133	Thigh	471	UPPER LEG (THIGH)
+Strong-1911-150-134	134	Thin	2308	THIN
+Strong-1911-150-135	135	Tongue	1205	TONGUE
+Strong-1911-150-136	136	Tooth	1380	TOOTH
+Strong-1911-150-137	137	Tree	906	TREE
+Strong-1911-150-138	138	Village	930	VILLAGE
+Strong-1911-150-139	139	Water	948	WATER
+Strong-1911-150-140	140	Weep	1839	CRY
+Strong-1911-150-141	141	White	1335	WHITE
+Strong-1911-150-142	142	Wife	1199	WIFE
+Strong-1911-150-143	143	Wind	960	WIND
+Strong-1911-150-144	144	Woman	962	WOMAN
+Strong-1911-150-145	145	Yam	410	YAM
+Strong-1911-150-146	146	One	1493	ONE
+Strong-1911-150-147	147	Two	1498	TWO
+Strong-1911-150-148	148	Three	492	THREE
+Strong-1911-150-149	149	Four	1500	FOUR
+Strong-1911-150-150	150	Five	493	FIVE
```

Comment:
Towards the end of the list, I think there are the following additional concepts:

- *Yes*
- *Yellow*

Reply:
oh, not sure how I missed them. Sorry, fixed.